getApiFilenames.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+
+const getApiFilenames = (dir, ignoredDirs = ['node_modules']) => {
+  const subdirs = fs.readdirSync(dir);
+  const docs = subdirs.map(subdir => {
+    const fileOrDir = path.join(dir, subdir);
+    return (fs.statSync(fileOrDir).isDirectory() ? getApiFilenames(fileOrDir, ignoredDirs) : getApiFile(fileOrDir, ignoredDirs));
+  });
+  return docs
+    .filter(file => file !== null)
+    .reduce((files, file) => files.concat(file), []);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat

```suggestion
    .flat();
```